modules/service/import-service.js

```diff
@@ -620,7 +620,9 @@ class ImportService {
         const otObjects = [];
 
         for (let i = 0; i < reconstructedObjects.length; i += 1) {
-            ImportUtilities.sortGraphRecursively([reconstructedObjects[i]]);
+            // TODO Use sortObjectRecursively here
```

Comment:
remove comment

Reply:
This is a placeholder for new sorting method which should be implemented in the next sprint